quipucords/scanner/acs/api.py

```diff
@@ -0,0 +1,50 @@
+"""ACS api adapter."""
+
+from logging import getLogger
+
+from requests.auth import AuthBase
+
+from compat.requests import Session
+
+logger = getLogger(__name__)
+
+
+class HTTPBearerAuth(AuthBase):
+    """A class for implementing HTTP Bearer Token Authentication."""
+
+    def __init__(self, auth_token):
+        self.auth_token = auth_token
+
+    def __eq__(self, other):
+        """Check if two instances have equal auth tokens."""
+        return self.auth_token == getattr(other, "auth_token", None)
+
+    def __ne__(self, other):
+        """Check if two instances have unequal auth tokens."""
+        return not self == other
```

Comment:
neat, but not necessary.

remember that every bit of code we add we have to maintain, so this is one of these cases where less is more.